book/src/pickers.md

```diff
@@ -6,6 +6,6 @@ Helix has a variety of pickers, which are interactive windows used to select var
 
 Most pickers perform fuzzy matching using [fzf syntax](https://github.com/junegunn/fzf?tab=readme-ov-file#search-syntax). Two exceptions are the global search picker, which uses regex, and the workspace symbol picker, which passes search terms to the LSP. Note that OR operations (`|`) are not currently supported.
 
-If a picker shows multiple columns, you may apply the filter to a specific column by prefixing the column name with `%`. Column names can be shortened to any prefix, so `%p`, `%pa` or `%pat` all mean the same as `%path`. For example, a query of `helix %p .toml$ !lang` in the global search picker searches for the term "helix" within files with paths ending in ".toml" but not including "lang".
+If a picker shows multiple columns, you may apply the filter to a specific column by prefixing the column name with `%`. Column names can be shortened to any prefix, so `%p`, `%pa` or `%pat` all mean the same as `%path`. For example, a query of `helix %p .toml: !lang` in the global search picker searches for the term "helix" within files with paths ending in ".toml" but not including "lang".
```

Comment:
```suggestion
If a picker shows multiple columns, you may apply the filter to a specific column by prefixing the column name with `%`. Column names can be shortened to any prefix, so `%p`, `%pa` or `%pat` all mean the same as `%path`. For example, a query of `helix %p .toml !lang` in the global search picker searches for the term "helix" within files with paths ending in ".toml" but not including "lang".
```

I would just leave out the colon. The colon has little impact. The period is already included so we are only matching the file extensions. The only thing that the colon adds is making `<path>.toml<bar>` rang lower than `<path>.toml`. That pretty niche and wold just add confusion what the colon means here

Reply:
I see, it makes indeed no difference in the examples I tried, thanks 👍